client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -58,9 +58,9 @@ module.exports = React.createClass( {
 
 		return (
 			<div className="plugin-meta__actions">
-				<PluginActivateToggle plugin={ this.props.plugin } site={ this.props.selectedSite } notices={ this.props.notices } />
-				<PluginAutoupdateToggle plugin={ this.props.plugin } site={ this.props.selectedSite } notices={ this.props.notices } wporg={ this.props.plugin.wporg } />
-				<PluginRemoveButton plugin={ this.props.plugin } site={ this.props.selectedSite } notices={ this.props.notices } />
+				<PluginActivateToggle { ...this.props } site={ this.props.selectedSite } />
```

Comment:
My preference is for writing out all the attributes. It is more to typing, but it is much easier to follow what is going on and what is being passed around. Since it is more explicit.